include/dab/select_cus_vcvc.h

```diff
@@ -0,0 +1,56 @@
+/* -*- c++ -*- */
+/* 
+ * Copyright 2017 <+YOU OR YOUR COMPANY+>.
```

Comment:
Since this file seems to be completely your work, your name should definitely be here.